caffe2/utils/CMakeLists.txt

```diff
@@ -1,15 +1,24 @@
-set(Caffe2_CPU_SRCS ${Caffe2_CPU_SRCS}
-        utils/proto_utils.cc
-        utils/murmur_hash3.cc
-        utils/smart_tensor_printer.cc
-        utils/signal_handler.cc
-        utils/string_utils.cc
-        utils/threadpool/ThreadPool.cc
-        utils/cpuid.cc
-        utils/bench_utils.cc
-        utils/math_cpu.cc
-        utils/math_utils.cc
-        )
+list(APPEND Caffe2_CPU_SRCS
+  utils/proto_wrap.cc)
+
+# ---[ only support the above when full caffe2 isn't built
+if (NOT BUILD_CAFFE2)
```

Comment:
Also not thrilled about this, but also will go away with https://github.com/pytorch/pytorch/pull/8338